be_hello.py

```diff
@@ -0,0 +1,23 @@
+#submitted by Keren Or Hadad 208025205
+
+def be_hello(input):
+    """
+
+    :param input: a text
+    :return: returns a list of the length of the words in the text
+    """
+    input = input.split(" ")
+    item = []
+    for x in input:
```

Comment:
Why not using list comprehension?

Reply:
This is not a complicated loop so I thought it would be better that way